### DISCUSSION TITLE
API architecture showdown - Rest vs graphQL vs gRPC

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#76] [BlogPosts]
# API architecture showdown - Rest vs graphQL vs gRPC

Three technologies stand out as primary choices for developing API’s REST, GraphQL, and gRPC. Understanding what they bring to the table and making sure it aligns with your API design goals is key in developing successful API’s

https://pradeepl.com/blog/api/rest-vs-graphql-vs-grpc/

**Comment by hackertaco** [comment]
I appreciate to read this. And I think this is very good comparison. Could I translate this into korean? Of course I would leave the link.

**Reply by PradeepLoganathan** [reply]
Thanks. Does the translate option in the browser not work well. I don't know korean so would do exactly what the browser based translate option provides.
